header-rewrite-filter/header_processor.cc

```diff
@@ -95,6 +94,71 @@ namespace HeaderRewriteFilter {
         request_headers->setPath(request_path); // should never return an error
     }
 
+
+    void SetBoolProcessor::setStringsToCompare(std::pair<absl::string_view, absl::string_view> strings_to_compare) {
+        std::string first_string = std::string(strings_to_compare.first);
+        std::string second_string = std::string(strings_to_compare.second);
+
+        strings_to_compare_ = std::make_pair(first_string, second_string); 
+    }
+
+    absl::Status SetBoolProcessor::parseOperation(std::vector<absl::string_view>& operation_expression) {
+        if (operation_expression.size() < Utility::SET_BOOL_MIN_NUM_ARGUMENTS) {
+            return absl::InvalidArgumentError("not enough arguments for set-bool");
+        }
+
+        try {
+            absl::string_view bool_name = operation_expression.at(2);
+            setBoolName(bool_name);
+
+            if (operation_expression.at(4) != "-m") {
+                return absl::InvalidArgumentError("invalid match syntax");
+            }
+
+            const Utility::MatchType match_type = Utility::StringToMatchType(operation_expression.at(5));
+
+            switch (match_type) {
+                case Utility::MatchType::Exact:
+                    {
+                        std::pair<absl::string_view, absl::string_view> strings_to_compare(operation_expression.at(3), operation_expression.at(6));
```

Comment:
This is getting the 7th (0-indexed) val, but SET_BOOL_MIN_NUM_ARGUMENTS is 5. Should `SET_BOOL_MIN_NUM_ARGUMENTS` be 7?